dbt_project.yml

```diff
@@ -1,5 +1,5 @@
 name: 'microsoft_ads'
-version: '0.4.0'
+version: '0.4.1'
```

Comment:
What is your reasoning for having this be a "patch" change rather than a "breaking" change?

Reply:
I mentioned it a bit earlier but to have it on record, I believe this should be a patch since it's adding logic that shouldn't affect the end user's experience. The logic is directly copied and pasted from the source package and changing the stg model var to a reference to an intermediate table instead.

Reply:
I completely agree with this reasoning. Thank!